src/azul/__init__.py

```diff
@@ -309,8 +309,8 @@ def _validate_term(self, term: str, name: str = 'Term'):
         require(self.term_re.fullmatch(term) is not None,
                 f"{name} is either too short, too long or contains invalid characters: '{term}'")
 
-    def google_service_account(self, lambda_name):
-        return f"dcp/azul/{self.deployment_stage}/{lambda_name}/google_service_account"
+    def secrets_manager_secret_name(self, *args):
+        return f"dcp/azul/{self.deployment_stage}/{'/'.join(args)}"
```

Comment:
```suggestion
        return '/'.join(['dcp', 'azul', self.deployment_stage, *args])
```

Is more expressive, less brittle (doesn't duplicate the separator) and as long (not counting whitespace, which increases readability).

Reply:
I like it better too.